docs/customization/repository.rst

```diff
@@ -67,29 +79,38 @@ As we are selecting Products we need to have a join to translations, because the
 We are sorting the results by the count of how many products are still available on hand, which is saved on the ``onHand`` field on the specific ``variant`` of each product.
 Then we are limiting the query to 8 by default, to get only 8 products that are low in stock.
 
-**2.** In order to use your repository you need to configure it in the ``config/_sylius.yaml``.
+**2.** In order to use your repository you need to configure it in the ``config/packages/_sylius.yaml``.
+As you can see in the ``_sylius.yaml`` you already have a basic configuration, now you just need to add your repository and override resourceRepository
 
 .. code-block:: yaml
 
     sylius_product:
         resources:
             product:
                 classes:
+                ...
                     repository: App\Repository\ProductRepository
```

Comment:
Shouldn't it be placed in `App\Doctrine\ORM\ProductRepository`? The interface in `Repository` namespace is good for me, as it's not coupled with ORM in any way.